ci/prow_setup.md

```diff
@@ -37,6 +37,13 @@
 
 1. Run `make update-boskos-config` to update the Boskos config.
 
+1. Increase the compute CPU quota for the project to 200. Go to
+   `https://pantheon.corp.google.com/iam-admin/quotas?project=<project_name>&service=compute.googleapis.com&metric=CPUs`
+   and click `Edit Quota`. This needs to be done manually and should be
```

Comment:
I clicked on `Edit Quota` in knative-boskos-01, and it asked `Select one or more quotas to edit.`

Reply:
you need to select the box next to it and then click on edit quota

Reply:
there are 20 boxes, each one represents different regions, I think we need to at least request quota for `us-central1`, `us-west1`, and `us-east1`

Reply:
up[dated to increase for 5 regions as we might run perf tests in 5 regions :)